source/_introduction.html.md.erb

```diff
@@ -298,9 +298,38 @@ The JSON representation of the objects returned from the List All REST endpoints
 Smartsheet will add new functionality and bug fixes to the API over time. Make sure that your code can handle new JSON properties gracefully. 
 Also, make sure your code does not depend on the order in which JSON objects are returned, unless it is explicitly stated in this documentation. 
 
+In some cases, such as when new column types or new widgets are added, Smartsheet offers a backwards-compatibility option as well as giving you access to the new functionality. For instance, if a sheet has column types of MULTI_CONTACT_LIST or MULTI_PICKLIST, there are two ways of receiving the response:
+
+* As text (column type TEXT_NUMBER)
+* As an array of values (column types of MULTI_CONTACT_LIST or MULTI_PICKLIST)
+
+Smartsheet lets you choose how and when to upgrade your queries and response values by using a "level" query parameter as a differentiator, for instance "?level=2". There are also incremented values in the Column object's "version" parameter. However, level and version vary by whether you are calling a report, sheet, or Sight (aka dashboard) and by what data you want to return.
+
+The read-only "version" parameter for the Column object increments based on the UI feature. You can see it in the response to various GET endpoints:
```

Comment:
I am not aware of a "version" parameter. Can you elaborate on what this is?

Reply:
'parameter' ==> 'property'

Reply:
Maybe it's a value? Not sure the right word. Here it is in a column object: "columns": [
        {
            "virtualId": 8660306940454788,
            "version": 0,
            "index": 0,
            "title": "Sheet Name",
            "type": "TEXT_NUMBER",
            "sheetNameColumn": true,
            "validation": false,
            "width": 150
        },

Reply:
Looks like you have used the word 'attribute' elsewhere.